PhysicsTools/PatAlgos/python/slimming/isolatedTracks_cfi.py

```diff
@@ -67,7 +70,14 @@
 from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
 pp_on_AA.toModify(isolatedTracks, useHighPurity = True)
 
+# full set of track extras not available in existing AOD
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
+
+(run2_miniAOD_80XLegacy | run2_miniAOD_94XFall17 | pp_on_AA).toModify(isolatedTracks, useSiPixelTrackProbQXY = False)
```

Comment:
```suggestion
from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
from Configuration.ProcessModifiers.run2_miniAOD_UL_preSummer20_cff import run2_miniAOD_UL_preSummer20

(run2_miniAOD_80XLegacy | run2_miniAOD_94XFall17 | run2_miniAOD_UL_preSummer20).toModify(isolatedTracks, useSiPixelTrackProbQXY = False)
```

   * `run2_miniAOD_UL_preSummer20` should be needed to address the remaining failing workflows
   * I am not sure why you need to exlclude `pp_on_AA` ?